devito/__init__.py

```diff
@@ -1,12 +1,16 @@
 from collections import namedtuple
 from itertools import product
 
+# Import the global `configuration` dict
+from devito.parameters import *  # noqa
+
 # API imports
 from devito.base import *  # noqa
 from devito.builtins import *  # noqa
 from devito.data.allocators import *  # noqa
 from devito.equation import *  # noqa
 from devito.finite_differences import *  # noqa
+from devito.mpi import MPI  # noqa
```

Comment:
Wait shouldn't this be a try catch?

Reply:
no, it's safe, see [here](https://github.com/opesci/devito/blob/master/devito/mpi/distributed.py#L33)